utils/archive.js

```diff
@@ -0,0 +1,48 @@
+const supabase = require("@supabase/supabase-js");
+
+module.exports = {
+  name: "taskArchive",
+  async execute() {
+    const client = supabase.createClient(
+      process.env.SUPABASE_URL,
+      process.env.SUPABASE_KEY
+    );
+
+    // fetch data
+    const { data, err } = await client
+      .from("tasks")
+      .select("guild_id, user_id, task_name, done")
+      .order("created_at", { ascending: true });
+    if (err) {
+      console.log("Fetch failed");
+      return;
+    }
+
+    // remove old data
+    await client.from("tasks").delete().in("done", [true, false]);
```

Comment:
Required: Adding error handling would be better.

Reply:
OK